### DISCUSSION TITLE
Add database flag to CLI to choose database when generating a new blitz project

### DISCUSSION THREAD

**Discussion by lhocke** [blitz-js/blitz#3668] [Ideas] [answered]
It's not crucial but it might be nice to be able to just specify which database a project will use when creating the project with something like --database="DATABASE" aliased as -d (possibly), which would generate an appropriate schema for Prisma with the options for that database already filled in, with a default of SQLite. It would provide flexibility for development using different databases while leaving the baseline experience intact.

**Comment by beerose** [comment, marked as answer]
Hey @lhocke, this is a good idea, but we haven't had time to work on it yet. Here's a related issue: https://github.com/blitz-js/blitz/issues/1037
